app/assets/javascripts/components/story/ExpandedStory/index.js

```diff
@@ -46,8 +47,8 @@ export class ExpandedStory extends React.Component {
         <ExpandedStoryControls
           onCancel={onToggle}
           isDirty={story._editing._isDirty || false}
-          onSave={() => saveStory(story.id, project.id)}
-          onDelete={() => deleteStory(story.id, project.id)}
+          onSave={() => saveStory(story.id, project.id, {}, from)}
```

Comment:
What is the empty `{}`? This action now has too many parameters in a way that makes it hard to understand what they are at the call site, we should find a way to make it clearer (but I need to know why the `{}` before suggesting something).

Reply:
The third param os action is `options`.
`export const saveStory = (storyId, projectId, options, from = defaultFrom) =>`

but in this case, we will not use this param.

Reply:
Oh, I see :thinking:, what do you think about moving the `options` argument to the last one so we don't need to specify it this way? I think it would make the code easier to understand (but notice it'll affect other places of the code and the tests, so we need to change those as well).

Reply:
Ok! I believe that will be more clear too